experiment/pretest.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "question": "1. The discharge of broad crested weir is maximum if the head of water on the downstream side of weir as compared to the head on the upstream side of is:",
+    "answers": {
+      "a": "two-third",
+      "b": "one-third",
+      "c": "one-half",
+      "d": "three-fourth"
+    },
+    "correctAnswer": "a"
+  },
+  {
+    "question": "2. Head loss will be small in case of:",
+    "answers": {
+      "a": "Ogee shaped",
+      "b": "Broad crested",
+      "c": "Sharp crested",
+      "d": "Narrow crested"
+    },
+    "correctAnswer": "b"
+  }
+]
```

Comment:
Pretest and posttest have the same questions, which does not make sense. Pretest is to measure the learner on the prerequisites of the experiment whereas posttest measures his learning from the experiment.